package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "cdmx009-mdlinks",
+  "version": "1.0.0",
+  "description": "\"Validate MarkDown Links\"",
```

tu descripción debe contener más información de lo que hace tu librería.
```suggestion
  "description": "\"Validate MarkDown Links and generate stats\"",
```